### PR TITLE
Restaurant page

### PR DESCRIPTION
**Pull request by SamArsanios**
In this project, I built a restaurant page that comprises different tabs(Home/Menu/Contact) using Javascript. A user is able to toggle between the  different tabs.

### CODE REVIEW

File: dist/style.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    background-color:#eee;
+  }
+  /* .card {
+    max-width: 540px;
+  }
+  
+  .card-img {
+    height: 100%;
+  } */
+
```
Comment (rukundoeric) [inline dist/style.css]: remove this comment

File: dist/style.css [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+.arsani {
+    text-align: center;
+    /* padding-top: 20px;
+    padding-bottom: 20px; */
```
Comment (rukundoeric) [inline dist/style.css]: remove this comment

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import home from './home';
+import menu from './menu';
+import contact from './contact';
+import header from './header';
+// import bgImg from './background-1.jpg';
+
```
Comment (rukundoeric) [inline src/index.js]: remove this comment

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+//   bg.style.background = `url(${bgImg})`;
+//   bg.style.backgroundRepeat = 'no-repeat';
+//   bg.style.backgroundSize = 'cover';
+
+//   main.appendChild(bg);
+//   main.appendChild(wrapper);
+//   content.appendChild(main);
+// };
+
+// main();
```
Comment (rukundoeric) [inline src/index.js]: remove this comments

**Review by rukundoeric — CHANGES_REQUESTED** [review]
# Status: ♻️ CHANGES REQUIRED ♻️
_Great job guys so far 👏 💯 👍. However, there are few issues we need to address before we move on, please check the comments below._

- ### To Highlight
  - [x] The code looks clean and professional. 💯
  - [x] The project follows Git flow. 💯

- ### To Improve
  - [x] [MANDATORY] Kindly remove all commented codes.
  - [x] [MANDATORY] Necessary run common commands like build, development should be in package.json inside `scripts'. Here is a basic example ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))
      ```json
        "scripts": {
          "dev": "webpack --mode development",
          "build": "webpack --mode production",
          "start": "webpack-dev-server --mode development --open" // optional
        },
      ```
  - [x] [MANDATORY] Fix the broken image in your readme. 

_Please make all the required changes before you submit another code review. If you have any problems or concerns, you can reach out to me on Slack: Rukundo Eric 🚀 . I wish you all the very best 🥇_

### _Keep working hard._
### _Happy Coding😃_
-----------------------------------------------------------------------------------------------------------------------------------------
**Regards,
[Rukundo Eric](https://rukundo-eric.herokuapp.com/) ,
[LinkedIn](https://www.linkedin.com/in/rukundo-eric/), 
Slack: Rukundo Eric**

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Samson Kibrom Okbay ** 
Good job with the improvements for this project :muscle:  :muscle:
It meets most of the requirements :clap: :clap: :clap:  
Just one last detail to improve before you can go ahead:

- [x] Fix broken iamge in your README file:



Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

**Review by rukundoeric — APPROVED** [review]
# Status: 🍾 PROJECT APPROVED 🍾
_Excellent job on this project guys. 🎆 Keep it up._

- ### To Highlight
  - [x] The project follows Git flow. 💯
  - [x] Code looks clean and professional 💯
  - [x] All required changes are implemented 💯

- ### To Improve
  - N/A

_Congratulations on the splendid job **@team** 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

### _Keep working hard._
### _Happy Coding😃_
-----------------------------------------------------------------------------------------------------------------------------------------
**Regards,
[Rukundo Eric](https://rukundo-eric.herokuapp.com/) ,
[LinkedIn](https://www.linkedin.com/in/rukundo-eric/),
Slack: Rukundo Eric**

**Pull request merged**